tests/conftest.py

```diff
@@ -3283,7 +3283,12 @@ def finalizer():
     logging_channel = "stable" if ocp_version >= version.VERSION_4_7 else ocp_version
 
     # Creates namespace openshift-operators-redhat
-    ocp_logging_obj.create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    try:
+        ocp_logging_obj.create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    except CommandFailed as e:
+        if "AlreadyExists" in str(e):
+            # on Rosa HCP the ns created from the deployment
+            log.info("Namespace openshift-operators-redhat already exists")
```

Comment:
Wouldn't make sense to re-raise the exception if it is not the `AlreadyExists` case?